src/pages/dongmin/Main/Feeds/Feeds.js

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import './Feeds.scss';
+import Comment from './Components/Comment';
+import { v4 as uuidv4 } from 'uuid';
+
+const Feeds = () => {
+  const [input, setInput] = useState('');
+  const [commentsList, setCommentsList] = useState([]);
+  const changeInput = e => {
+    setInput(e.target.value);
+  };
+
+  const deleteComment = idxOfBeingDeleted => {
+    setCommentsList(
+      commentsList.filter((el, idx) => {
+        return idx !== idxOfBeingDeleted;
+      })
+    );
+  };
+
+  const onSubmit = e => {
+    const comment = { name: 'gmgmgun', content: input };
+    setCommentsList(commentsList.concat(comment));
```

Comment:
concat도 좋은 방식이지만 아래와 같은 방식이 조금 더 잘 읽힐 것 같은데 어떻게 생각하시나요?
```suggestion
    setCommentsList([...commentsList, comment]);
```

Reply:
스프레드 연산자보다 mutator 메서드가 더 익숙해서 ^^; 가독성은 스프레드 연산자가 더 좋은 것 같습니다. 수정하면서 스프레드 연산자에 더 익숙해져보겠습니다!

Reply:
concat 도 immutable한 메서드입니다. 
추가 하는 상황에서 mutable한 메서드는 push 입니다.

<img width="729" alt="스크린샷 2023-01-31 오후 1 32 32" src="https://user-images.githubusercontent.com/53929065/215665953-adf15bca-296d-497b-b751-ace5e5ef3c76.png">

Reply:
아 맞습니다! immutable이라 push보다 concat을 사용하는 게 더 좋다고 해서 사용했는데 착각하고 있었네요 ㅜㅜ 감사합니다!